OpenSim/Simulation/Model/PhysicalFrame.cpp

```diff
@@ -45,11 +45,16 @@ PhysicalFrame::PhysicalFrame() : Frame()
 {
     setAuthors("Matt DeMers, Ayman Habib, Ajay Seth");
     constructProperties();
+    FrameGeometry* frm = new FrameGeometry(0.2);
```

Comment:
This FrameGeometry object gets leaked at the end of this method. Since it is *copied* to the GeometrySet property, it shouldn't be heap allocated here.